webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -37,8 +37,8 @@ const getNotifications = url => dispatch => {
     });
   }
 
-  function onGetNotificationsFailed(error) {
-    if (error.response.status === 401) {
+  function onGetNotificationsFailed({ response }) {
+    if (response && response.status === 401) {
```

Comment:
in which condition does this happens? does it make sense to add a test?

Reply:
I saw it once while moving between pages,
not sure if it was a problem in my stack or not because I couldn't reproduce..
but it seem like a good thing to nitpick ;)